apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/navigation-menu/index.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * WordPress dependencies
+ */
+import { registerBlockType } from '@wordpress/blocks';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import edit from './edit';
+import './style.scss';
+
+const icon = (
+	<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
+		<path fill="none" d="M0 0h24v24H0V0z" />
+		<path d="M12 7.27l4.28 10.43-3.47-1.53-.81-.36-.81.36-3.47 1.53L12 7.27M12 2L4.5 20.29l.71.71L12 18l6.79 3 .71-.71L12 2z" />
+	</svg>
+);
+
+registerBlockType( 'a8c/navigation-menu', {
+	title: __( 'Navigation Menu' ),
+	description: __( 'Visual placeholder for site-wide navigation and menus.' ),
+	icon,
+	category: 'layout',
+	supports: {
+		html: false,
+		multiple: false,
+		reusable: false,
+	},
+	attributes: {
+		themeLocation: {
+			type: 'string',
+			default: 'main-1',
+		},
+	},
+	edit,
+	save: () => null,
```

Comment:
This line should not be needed given [this is the default behavior when registering a block](https://github.com/WordPress/gutenberg/blob/a57202e635ff0cd190a09128d27951620db9a870/packages/blocks/src/api/registration.js#L77).

Reply:
Hm, weird; on removing it, I get a console error I can't quickly find: `The "save" property must be specified and must be a valid function.`

Reply:
Oh, then I'm certainly missing something else 😅 Disregard my comment then 😄